openwisp_network_topology/tests/test_websockets.py

```diff
@@ -20,15 +20,8 @@ class TestTopologySockets(CreateGraphObjectsMixin, CreateOrgMixin):
     topology_model = Topology
     application = import_string(getattr(settings, 'ASGI_APPLICATION'))
 
-    async def _get_url(self, topology_id, admin_view):
-        non_admin_view_url = f'topology/topology/{topology_id}/'
-        admin_view_url = f'admin/topology/topology/{topology_id}/change/'
-        if not admin_view:
-            return non_admin_view_url
-        return admin_view_url
-
     async def _get_communicator(self, admin_client, topology_id, admin_view=True):
-        url = await self._get_url(topology_id, admin_view)
+        url = f'admin/topology/topology/{topology_id}/change/'
```

Comment:
what if we use a URL which doesn't mention admin here? A more neutral naming I think is appropriate.
Something like `network-topology/topology/{pk}/`?

Reply:
sure